ley.config.js

```diff
@@ -1,9 +1,11 @@
+import postgres from 'postgres';
 import { setEnvironmentVariables } from './util/config.js';
 
 setEnvironmentVariables();
 
 const options = {
   ssl: Boolean(process.env.POSTGRES_URL),
+  transform: postgres.camel,
```

Comment:
or @Eprince-hub, should we share this code somehow? (instead of duplicating)

eg. also in `util/config.js` or something, and imported into `connect.ts` as well?

Reply:
oh it looks like we also lost the `ssl` Postgres.js config from Fall 2023:

- https://github.com/upleveled/next-js-example-fall-2023-atvie/blob/main/database/connect.ts

I'll add this to the PR

and it seems like this should indeed be shared between the two files?

Reply:
You mean this `ssl: Boolean(process.env.POSTGRES_URL),` ?

Reply:
yes

Reply:
> and it seems like this should indeed be shared between the two files?

I think this is a good idea

Reply:
and maybe we should use this `undefined: null` config as well (so that the behavior with `undefined` is consistent also with Ley):

https://github.com/upleveled/next-js-example-winter-2024-atvie/blob/3734bc3574cdc21c3664b976bd9ea2c79bd0d0cb/database/connect.ts#L16-L22

Reply:
@Eprince-hub done in 7c3b5de, please take a look

if you think it's good, can you also create a PR in the lecture notes for the multiple database + testing lectures for adding the config variable? (think about A) when will it be first added and B) when will it be extracted to `config/util.js` - maybe in the same step, but maybe not)

Reply:
@Eprince-hub done in 99ea339, please take a look

if you think it's good, can you also create a PR in the lecture notes for the multiple database + testing lectures for adding the config variable? (think about A) when will it be first added and B) when will it be extracted to `config/util.js` - maybe in the same step, but maybe not)

Reply:
A new PR has been created for this update
https://github.com/upleveled/courses/pull/2404